apps/js-machine-front/src/app/components/NavDesktop.tsx

```diff
@@ -1,30 +1,47 @@
-import React, { memo } from 'react';
+import React from 'react';
 import { NavLink } from 'react-router-dom';
 import { FormattedMessage } from 'react-intl';
 import { makeStyles } from '@material-ui/core/styles';
+import { useStores } from '@js-machine-app/front/stores';
+import { observer } from "mobx-react-lite";
 
 const useStyles = makeStyles(theme => ({
-  link: {
-    textDecoration: 'none',
-    color: theme.palette.primary.contrastText,
-    fontWeight: 'bold',
-    marginLeft: 60,
-  },
-  activeNavLink: {
-    // color should be in theme
-    color: '#f2e14c',
-  },
-}));
+    navDesktop: {
+      padding: '10px 20px',
```

Comment:
for padding you can use theme.spacing(1) (number parameter 1 is equal 8px; 2 - 16px)

Reply:
btw, you can use common class for example navDesktop with padding, and apply additional class for withBackground